plugins/rule_engines/irods_rule_engine_plugin-irods_rule_language/functions.cpp

```diff
@@ -2048,6 +2051,19 @@ Res *smsi_substr( Node** paramsr, int, Node* node, ruleExecInfo_t*, int, Env*, r
     return retres;
 }
 
+Res *smsi_error( Node** paramsr, int, Node* node, ruleExecInfo_t*, int, Env*, rError_t* errmsg, Region* r ) {
+	Res * errName = paramsr[0];
+	const auto& name_map = irods_error_map_construction::irods_error_name_map;
+	int i {};
+	try { i = name_map.at( errName->text ); } catch(...) { }
+	if ( i == int{} )
```

Comment:
what does using `int{}` give us over `0`?  this feels like it can confuse people down the road.

Reply:
Yes, agreed. changing as suggested.